packages/overlays/test/GlobalOverlayController.test.js

```diff
@@ -111,34 +115,6 @@ describe('GlobalOverlayController', () => {
       expect(ctrl.isShown).to.equal(false);
     });
 
-    it('puts the latest shown overlay always on top', async () => {
```

Comment:
Why did this one get removed?

Reply:
This test is a duplicate of: https://github.com/ing-bank/lion/blob/feat/bottomsheetRebased/packages/overlays/test/OverlaysManager.test.js#L84 , but was failing. I double checked the code, which worked. I couldn't get the test working, within a certain time frame, and since it's a duplicate i removed it.